code-gen-projects/typescript/code-gen-demo/src/generated/enumType.ts

```diff
@@ -0,0 +1,51 @@
+import * as ion from 'ion-js';
+import { IonSerializable } from './ion_generated_code';
+
+
+
+export enum EnumType {FOO_BAR_BAZ = "FooBarBaz", BAR = "bar", BAZ = "baz", FOO = "foo"
```

Comment:
I understand you submitting these generated files for making it easy for the reviewer to look at it but we generally don't merge the generated code int the repository. You can probably just create some gists and add it to the PR description.